config/runtime/server_internal_test.go

```diff
@@ -37,135 +36,6 @@ func TestServer_isUnique(t *testing.T) {
 	}
 }
 
-func TestServer_validatePortHosts(t *testing.T) {
```

Comment:
I am missing related test replacements. Wildcard and/or with multi hosts.

Reply:
Done